src/quick-lint-js/parse-expression-inl.h

```diff
@@ -1779,15 +1779,6 @@ expression* parser::parse_index_expression_remainder(Visitor& v,
   return this->make_expression<expression::index>(lhs, subscript, end);
 }
 
-template <QLJS_PARSE_VISITOR Visitor>
-expression* parser::parse_arrow_function_body(Visitor& v,
-                                              function_attributes attributes,
-                                              const char8* parameter_list_begin,
-                                              bool allow_in_operator) {
-  return this->parse_arrow_function_body_impl(
-      v, attributes, parameter_list_begin, allow_in_operator);
-}
-
 template <QLJS_PARSE_VISITOR Visitor>
 expression* parser::parse_arrow_function_body(
```

Comment:
This function is now useless. Inline `parse_arrow_function_body_impl` into this function.

Reply:
Solved this in the last uploaded commit https://github.com/quick-lint/quick-lint-js/pull/680/commits/1c488ed74bc8529521ee132c59122d98152563ef and also removed the default parameters in the parse_arrow_function_body function declaration